src/Microsoft.PowerShell.Commands.Management/commands/management/WMIHelper.cs

```diff
@@ -162,7 +162,7 @@ internal override void StartOperation()
             }
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-linux`
ConsoleHost unit tests.ApartmentState tests.Should fail to set apartment state to: -sta
```powershell
Expected 64, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 736
736:             $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
```
ConsoleHost unit tests.ApartmentState tests.Should fail to set apartment state to: -mta
```powershell
Expected 64, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 736
736:             $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
```